recipes/paml/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: paml
+  version: "4.9c"
+
+build:
+  number: 0
+
+source:
+  fn: paml-4.9c.tar.gz
+  url: http://abacus.gene.ucl.ac.uk/software/paml4.9c.tgz
+  md5: 852d60dd435bc3b1ce9c47c2a7b83806
+
+requirements:
+  build:
+    - gcc # [linux]
+
+# codeml binary does not accept arguments, so any mock execution produces an
+# exit code !=0. a quick real execution is therefore used as a test case.
```

Comment:
Does it return anything useful as text beside this exit code?
If you you could do this: `codeml | grep "useful text"`